paper-tab.html

```diff
@@ -116,6 +116,18 @@
         Polymer.PaperRippleBehavior
       ],
 
+      /**
+       * Fired when the paper-tab recieves focus.
```

Comment:
Thanks, this is good to know; hopefully I should be able to get rid of the events altogether.